### PR TITLE
[Stewart] Add block patterns

### PR DESCRIPTION
**Pull request by kjellr**
This PR adds 16 block patterns to Stewart.

Slice 1

Header: 
- Header Image
- Traditional header
- Traditional header with color background

Footer:
- Default footer
- Left-aligned footer
- Right-aligned footer
- Footer with left navigation
- Footer with right navigation

Sidebar: 
- Default sidebar
- Sidebar with border
- Sidebar with background color
- Sidebar with introduction
- Blogging sidebar
- Sidebar with categories and tags

Page: 
- About
- Contact

### CODE REVIEW

File: stewart/parts/header-title-nav.html [hunk 1]
```diff
@@ -1,6 +1,11 @@
...
+<!-- wp:group {"align":"full","layout":{"inherit":true}} -->
+<div class="wp-block-group alignfull"><!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"var(-d-dwp-d-dcustom-d-dmargin-d-dhorizontal, 30px)","bottom":"40px"}}},"layout":{"type":"flex","justifyContent":"space-between"}} -->
+<div class="wp-block-group alignwide" style="padding-top:var(--wp--custom--margin--horizontal, 30px);padding-bottom:40px"><!-- wp:group {"layout":{"type":"flex"}} -->
+<div class="wp-block-group"><!-- wp:site-logo {"width":64} /-->
+
+<!-- wp:site-title /--></div>
+<!-- /wp:group -->
+
+<!-- wp:navigation {"__unstableLocation":"primary","layout":{"type":"flex","orientation":"horizontal"}} /--></div>
+<!-- /wp:group --></div>
```
Comment (kjellr) [inline stewart/parts/header-title-nav.html:10]: This template was edited to ensure that this version of the header is wide-aligned, like the rest of the site content. It's in sync with the "Traditional header" pattern.

File: stewart/parts/footer.html [hunk 2]
```diff
@@ -1,15 +1 @@
...
-	<!-- wp:paragraph {"align":"center","fontSize":"tiny"} -->
-	<p class="has-text-align-center has-tiny-font-size">Proudly Powered by <a href="https://wordpress.org" rel="nofollow">WordPress</a></p>
-	<!-- /wp:paragraph -->
-</div>
-<!-- /wp:group -->
-
-<!-- wp:spacer {"height":60} -->
-<div style="height:60px" aria-hidden="true" class="wp-block-spacer"></div>
-<!-- /wp:spacer -->
+<!-- wp:pattern {"slug":"stewart/footer-default"} /-->
```
Comment (kjellr) [inline stewart/parts/footer.html:1]: The footer template now uses this pattern to avoid untranslatable strings.

**Comment by kjellr** [comment]
Made a small revision to the "About" page pattern. The content below the picture now looks like this: 

Screen Shot 2022-01-20 at 9 31 42 AM

File: stewart/inc/patterns/sidebar-border.php [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+					<!-- wp:site-title {"textAlign":"center","style":{"typography":{"lineHeight":"1.2"}},"fontSize":"gigantic"} /-->
+
+					<!-- wp:site-tagline {"textAlign":"center","fontSize":"small"} /-->
+
+					<!-- wp:separator {"className":"is-style-wide"} -->
+					<hr class="wp-block-separator is-style-wide"/>
+					<!-- /wp:separator -->
+
+					<!-- wp:navigation {"ref":99,"overlayBackgroundColor":"background","overlayTextColor":"foreground","layout":{"type":"flex","setCascadingProperties":true,"justifyContent":"center","orientation":"vertical"},"style":{"spacing":{"blockGap":"0px"}}} /-->
```
Comment (pbking) [inline stewart/inc/patterns/sidebar-border.php]: I think this "ref":99 is causing a "menu has been deleted or is unavailable" error

image

File: stewart/inc/patterns/sidebar-background.php [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+					<div class="wp-block-group" style="padding-top:0em;padding-right:2em;padding-bottom:2em;padding-left:2em"><!-- wp:site-title {"textAlign":"left","style":{"typography":{"lineHeight":"1.2"}},"fontSize":"gigantic"} /-->
+
+					<!-- wp:site-tagline {"textAlign":"left","fontSize":"small"} /-->
+
+					<!-- wp:separator {"className":"is-style-wide"} -->
+					<hr class="wp-block-separator is-style-wide"/>
+					<!-- /wp:separator -->
+
+					<!-- wp:navigation {"ref":99,"overlayBackgroundColor":"background","overlayTextColor":"foreground","layout":{"type":"flex","setCascadingProperties":true,"justifyContent":"left","orientation":"vertical"},"style":{"spacing":{"blockGap":"0px"}}} /-->
+
```
Comment (pbking) [inline stewart/inc/patterns/sidebar-background.php:25]: I think this "ref" should also be removed.

Comment (kjellr) [reply stewart/inc/patterns/sidebar-background.php:25]: ✅

File: stewart/inc/patterns/sidebar-border.php (hunk 3 above)
Comment (kjellr) [reply stewart/inc/patterns/sidebar-border.php]: Ah, good catch. Should be fixed in 5ac6604.

**Review by pbking — APPROVED** [review]
LGTM

**Pull request merged**